pypeit/specobj.py

```diff
@@ -559,6 +559,28 @@ def to_xspec1d(self, **kwargs):
         # Create
         return xspectrum1d.XSpectrum1D.from_tuple((wave, flux, sig))
 
+    def vette_for_extraction(self):
```

Comment:
I think anything required for extraction should be part of the output data model, which I think means adding the lowercase tags here to the data model and making them upper case. 

I'm just imagining a situation where I want to read in a specobjs object and perform extraction again in a different way via some afterburner. It is not something we support, but I could imagine adding it.

Reply:
yes, that is sensible.